app/styleguide/routes/product-detail.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div :class="b()">
```

Comment:
- [ ] Please add `height: 100%` so the page height does not collapse. This is a workaround for the Styleguide, where the main content is wrapped inside the router container.

Reply:
As discussed I added `100vh` to the s-layout component.